src/components/organisms/MessageBox/MessageBox.tsx

```diff
@@ -27,6 +27,20 @@ const MessageBox: React.FC = () => {
     [alert]
   );
 
+  const modify = (title: string) => {
+    const wordsArray = title.split(' ');
+    const unauthIndex = wordsArray.indexOf('UNAUTHORIZED');
```

Comment:
You don't have to split the text into words because there might be a case where we have `[Unauthorized]` in the text and then we will not catch that.

I would instead do the following:

```ts
const isUnauthorizedError = title.toLowerCase().includes('unauthorized');
```